docs/app.js

```diff
@@ -0,0 +1,284 @@
+import mermaid from 'https://cdn.jsdelivr.net/npm/mermaid@10/dist/mermaid.esm.min.mjs';
+import { marked } from 'https://cdn.jsdelivr.net/npm/marked@11/lib/marked.esm.js';
+
+// Configure marked for GitHub-flavored markdown
+marked.setOptions({
+    gfm: true,
+    breaks: true
+});
+
+// Documentation sections configuration
+const documentationSections = [
+    {
+        id: 'overview',
+        title: 'Overview',
+        type: 'custom'
+    },
+    {
+        id: 'contributing',
+        title: 'Contributing',
+        file: '../CONTRIBUTING.md',
```

Comment:
would adding symlinks to `docs` work here? we could then just iterate over that symlink folder and assume each "folder" is a group and assume the title from the readme title (?)

"Overview" would be the exception.